### DISCUSSION TITLE
[Nextcloud] Is it possible to name the first account logged into?

### DISCUSSION THREAD

**Discussion by MinamiHasaki** [the-djmaze/snappymail#1194] [Q&A]
Omg, is SnappyMail so much faster than the original mail client app.

When logging into the first email account, you provide the email and password for the account. When adding other accounts, you provide email, password, and a name. The name is shown in the dropdown when selecting which account is accessed. Is there a way to name the first account logged into? In the general settings, there is a section to update the identify, but that looks to only apply to the message body and not the dropdown.

**Comment by cm-schl** [comment]
I'm not 100% sure but as far as i know this is not possible because SnappyMail internally makes a difference between additional accounts and the main account. I think that I saw some time ago that the main account has no name variable inside it's class...

**Reply by MinamiHasaki** [reply]
It looks like from here that they share the same properties: [snappymail/v/0.0.0/app/libraries/RainLoop/Model/MainAccount.php](https://github.com/the-djmaze/snappymail/blob/master/snappymail/v/0.0.0/app/libraries/RainLoop/Model/MainAccount.php)

**Reply by cm-schl** [reply]
You're right.
I can't remember exactly, but I wanted to do the same thing when writing my plugin https://github.com/cm-schl/snappymail/tree/master/plugins/ldap-mail-accounts and did not find a solution to this... But maybe there is some way to do this I don't know yet 🙂.
